tests/embed.spec.ts

```diff
@@ -65,3 +69,29 @@ test('can click and delete image', async ({ page }) => {
   await redoByKeyboard(page);
   await assertRichImage(page, 0);
 });
+
+test('press enter will create new block when click and select image', async ({
+  page,
+}) => {
+  await enterPlaygroundRoom(page);
+  await initImageState(page);
+  await assertRichImage(page, 1);
+
+  await activeEmbed(page);
+  await page.keyboard.press('Enter', { delay: 50 });
+  await page.keyboard.type('aa');
+  await assertRichTexts(page, ['\n', 'aa']);
+});
+
+test('hover image block will show image-option', async ({ page }) => {
```

Comment:
Please refer to https://github.com/toeverything/blocksuite/pull/310#discussion_r1049442419 about styling on the grouping of test cade code. Tight continuous spaghetti codes are not preferred.